cmd/git/git.go

```diff
@@ -1026,167 +1028,76 @@ func (j *DSGit) SetParentCommitFlag(richItem map[string]interface{}) (err error)
 }
 
 // GetModelData - return data in swagger format
-func (j *DSGit) GetModelData(ctx *shared.Ctx, docs []interface{}) (data *models.Data) {
-	data = &models.Data{
-		DataSource: GitDataSource,
```

Comment:
`DataSource: GitDataSource` entire `GitModelData` and `gGitMetaData` should be removed, basically anything that comes from swagger and swagger itself should be removed.

Reply:
done